nilearn/plotting/img_plotting.py

```diff
@@ -62,6 +63,18 @@ def _plot_img_with_bg(img, bg_img=None, cut_coords=None,
         display_factory: function
             takes a display_mode argument and return a display class
     """
+    show_nan_msg = False
+    if ('vmax' in kwargs and np.isnan(kwargs['vmax'])):
```

Comment:
`if ('vmax' in kwargs and kwargs['vmax'] is not None and np.isnan(kwargs['vmax'])):`